docs/install/ref_configs.rst

```diff
@@ -33,7 +33,7 @@ To generate the example configurations run the following from the root of the re
 .. code-block:: console
 
   mkdir -p generated/configs
-  configs/configgen.sh generated/configs
+  python ./configs/configgen.py generated/configs
```

Comment:
Yeah, there's been some drift here during the Bazel conversion, thanks for bringing this to our attention and the fix. I think a perhaps cleaner way to do this is to replace all the instructions above with:

```
mkdir -p generated/configs
bazel build //configs:example_configs
tar xvf $PWD/bazel-genfiles/configs/example_configs.tar -C generated/configs
```

The advantage is that the generation is captured by the Bazel build process, which is caught by tests in regression so we avoid future drift.

@jjwilcoxson Do you want to verify the above flow works for you and update the PR? Appreciated.

@mattklein123 We don't need CLA for docs right?